internal/collections/named.go

```diff
@@ -94,11 +94,11 @@ func (c *NamedCollection) String() string {
 }
 
 func (c *NamedCollection) addName(key string) {
-	for _, n := range c.names {
-		if n == key {
-			return
-		}
-	}
+	// for _, n := range c.names {
```

Comment:
This breaks the assumption that `NamedCollection` contains only unique values, put would permit `921170` to count them

Reply:
Interesting we did not break any test by commenting this out. Is there any case we need unique values? @jptosso @fzipri @piyushroshan

Reply:
IMHO, uniqueness was causing issues, especially to figure parameter pollution etc.